recipe/meta.yaml

```diff
@@ -220,14 +223,14 @@ outputs:
         - test -f "${PREFIX}/lib/libquadmath.dylib"    # [not (osx and arm64)]
         - test -f "${PREFIX}/lib/libquadmath.0.dylib"  # [not (osx and arm64)]
 
-        - test -f "${PREFIX}/lib/libgcc_s.1.dylib"          # [target_platform == "osx-64" and gfortran_version != "11.2.0"]
-        - test -f "${PREFIX}/lib/libgcc_s_ppc64.1.dylib"    # [target_platform == "osx-64" and gfortran_version != "11.2.0"]
-        - test -f "${PREFIX}/lib/libgcc_s_x86_64.1.dylib"   # [target_platform == "osx-64" and gfortran_version != "11.2.0"]
+        - test -f "${PREFIX}/lib/libgcc_s.1.dylib"          # [target_platform == "osx-64" and gfortran_version != "11.3.0"]
+        - test -f "${PREFIX}/lib/libgcc_s_ppc64.1.dylib"    # [target_platform == "osx-64" and gfortran_version != "11.3.0"]
+        - test -f "${PREFIX}/lib/libgcc_s_x86_64.1.dylib"   # [target_platform == "osx-64" and gfortran_version != "11.3.0"]
 
-        - test -f "${PREFIX}/lib/libgcc_s.1.1.dylib"        # [target_platform == "osx-arm64" or gfortran_version == "11.2.0"]
+        - test -f "${PREFIX}/lib/libgcc_s.1.1.dylib"        # [target_platform == "osx-arm64" or gfortran_version == "11.3.0"]
 
 about:
-  home: http://gcc.gnu.org/
+  home: https://gcc.gnu.org/
   license: GPL-3.0-only WITH GCC-exception-3.1
```

Comment:
I think the license should be `GPL-3.0-or-later`, see https://gcc.gnu.org/onlinedocs/gcc-11.3.0/gfortran/Copying.html#Copying. 
I didn't find GCC Runtime Library Exception, see the text https://www.gnu.org/licenses/gcc-exception-3.1.html